spec/tic_tac_toe_spec.rb

```diff
@@ -0,0 +1,103 @@
+# spec/board_spec.rb
+
+require './lib/board'
+require './lib/game'
+require './lib/player'
+
+describe Board do
+  subject { Board.new(order) }
+  before { @player1 = Player.new('Player1', 'X') }
+  before { @player2 = Player.new('Player2', '0') }
+  before { @game = Game.new([@player1, @player2], 3) }
```

Comment:
- [ ] Ensure you use `let` to define a variable in your test case instead of `before` like you have it. Here is a [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/Ruby_testing/before_each.png) from Pathwright.